trace/trace_quickstart/main.go

```diff
@@ -77,7 +77,10 @@ func main() {
 		port = "8080"
 	}
 	log.Printf("Listening on port %s", port)
-	if err := http.ListenAndServe(":"+port, nil); err != nil {
+	if err := http.ListenAndServe(":"+port, &ochttp.Handler {
+        // Use Google Cloud propagation format.
+        Propagation: &propagation.HTTPFormat{},
+        }); err != nil {
```

Comment:
Would you mind declaring a separate variable for the `ochttp.Handler`? Some developers have had issues with the inline calling of `ListenAndServe`, so probably good to not make the line more complex.

Might also be good to add a comment saying what this results in.